go.mod

```diff
@@ -133,6 +133,7 @@ replace (
 	github.com/openshift/hive => github.com/openshift/hive v1.1.17-0.20220223000051-b1c8fa5853b1
 	github.com/openshift/hive/apis => github.com/openshift/hive/apis v0.0.0-20220221165319-b389a65758da
 	github.com/portworx/sched-ops => github.com/portworx/sched-ops v0.20.4-openstorage-rc3
+	github.com/red-hat-storage/ocs-operator/api/v4 => github.com/rewantsoni/ocs-operator/api/v4 v4.0.0-20240701052137-de69df292a5d
```

Comment:
To be removed as soon as this merges https://github.com/red-hat-storage/ocs-operator/pull/2466

Reply:
Update: https://github.com/red-hat-storage/ocs-operator/pull/2677 is the PR, the older PR was before the design changes